configure.ac

```diff
@@ -201,6 +202,7 @@ AM_COND_IF([ENABLE_TOOLS], [
 
 AM_COND_IF(ENABLE_PROXY_SERVER, [CPPFLAGS+=" -DOPENDHT_PROXY_SERVER"], [])
 AM_COND_IF(ENABLE_PROXY_CLIENT, [CPPFLAGS+=" -DOPENDHT_PROXY_CLIENT"], [])
+AM_COND_IF(PROXY_CLIENT_OR_SERVER, [CPPFLAGS+=" -DPROXY_CLIENT_OR_SERVER"], [])
```

Comment:
never used in source code

Reply:
@aberaud Oh I thought I needed it here [src/Makefile.am#L75](https://github.com/binarytrails/opendht/blob/proxy_autotools/src/Makefile.am#L75) to enter this conditional

Reply:
Done at ad695035aa25ff725c51447a3aa5fb5241e9cfe2